src/visual.ts

```diff
@@ -1587,7 +1594,9 @@ module powerbi.extensibility.visual {
             textPathSelection
                 .attr({
                     startOffset: "50%",
-                    href: (link: SankeyDiagramLink) => `#linkLabelPaths${(link.source.label.name || "").replace(/\W*/g,"")}-${(link.destination.label.name || "").replace(/\W*/g,"")}`
+                    href: (link: SankeyDiagramLink) => {
+                        return SankeyDiagram.createLink(link, true);
```

Comment:
Change to 
return "#" + SankeyDiagram.createLink(link, true);
#is required to select by id